### PR TITLE
Embed Hubspot in subscribe-form potential

### PR DESCRIPTION
**Pull request by karlstolley**
We've been running this in production.

@lazeratops's feedback about the "Work Email" input should (?) be changed by Erik. I'm still seeing it in staging, but that's outside the scope of this PR. The bigger thing seems to be getting this deployed so Erik's newsletter work is unblocked.

### CODE REVIEW

**Comment by karlstolley** [comment]
Linear ticket [DEV-1534](https://linear.app/dailyco/issue/DEV-1534/blog-platform-update-newsletter-form-to-point-to-hubspot)

**Comment by karlstolley** [comment]
Linear ticket DEV-1534

**Review by lazeratops — APPROVED** [review]
LGTM (assuming by running this in production you actually meant staging, since I don't see it live in production!)

**Comment by karlstolley** [comment]
Oops. Yes, I meant staging 🙃 Thanks @lazeratops!

File: partials/subscribe-form.hbs [hunk 1]
```diff
@@ -8,57 +8,26 @@
...
+          }
+          </style>
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
+          <script>
+           hbspt.forms.create({
+           region: "na1",
+           portalId: "22536638",
+           formId: "89b9290a-e660-43a2-868f-e03f29eb3610",
+           version: "V2_PRERELEASE"
+           });
```
Comment (jayne-mast) [inline partials/subscribe-form.hbs:29]: Nitpicking, but the indentation is a bit off here

File: partials/subscribe-form.hbs [hunk 2]
```diff
@@ -8,57 +8,26 @@
...
-              class="js-cm-submit-button sc-jKJlTe kvTcQK"
-            >Subscribe</button>
-          </form>
+          <style>
+          /* Correctly style subscription confirmation in Firefox */
+          @-moz-document url-prefix() {
+            #site-main.site-main .hbspt-form iframe { min-height: 35px !important; }
+          }
+          </style>
+          <script charset="utf-8" type="text/javascript" src="//js.hsforms.net/forms/v2.js"></script>
```
Comment (jayne-mast) [inline partials/subscribe-form.hbs]:
```suggestion
          <script src="//js.hsforms.net/forms/v2.js"></script>
```

File: partials/subscribe-form.hbs [hunk 3]
```diff
@@ -8,57 +8,26 @@
       <div class="sc-bdVaJa iIDDUy">
         <div>
           <div>
-            <p size="16px" color="#434d5d" class="sc-eNQAEJ fBOZaM">Building on
-              video? Subscribe to our newsletter for all the latest product
-              releases, tutorials, and more from Daily.
+            <p size="16px" color="#434d5d" class="sc-eNQAEJ fBOZaM">
```
Comment (jayne-mast) [inline partials/subscribe-form.hbs:11]: Is `size` and `color` a thing in Handlebars? And what about all these hard-coded class names that look generated?

**Review by jayne-mast — APPROVED** [review]

File: partials/subscribe-form.hbs (hunk 3 above)
Comment (karlstolley) [reply partials/subscribe-form.hbs:11]: We'll clean that up on a subsequent PR: DEV-1542

**Pull request merged**